### DISCUSSION TITLE
Website blocked by antivirus

### DISCUSSION THREAD

**Discussion by 2nPlusOne** [react-hook-form/react-hook-form#11974] [General]
Basically the title. The react hook form website is suddenly getting blocked by our work antivirus solution.

```
Symantec Endpoint Protection blocked this website:
https://www.beekai.com/marketing/logo/horizontal.svg
```

![image](https://github.com/react-hook-form/react-hook-form/assets/39078738/837ce04a-8823-4742-a061-6b502bfb8d0d)

**Comment by yevhen309** [comment]
The same situation with  G DATA WebProtection antivirus. It seems the problem is with https://www.beekai.com/

**Comment by jle-pass** [comment]
Hi, same result on VirusTotal:
https://www.virustotal.com/gui/url/5a5e416f481357882bc4994b4d52b990de72e056b87675180864ce5e8b31012b

Actually, the svg file looks fine, unless the downloaded content depends on the client IP address requesting it !?

After site contact, their response is:
> There are few bad actors used BEEKAI (form builder) platform for phishing but banned as soon as getting reported.
